component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAPIHandler.java

```diff
@@ -51,122 +47,90 @@ public class BackupCodeAPIHandler {
      * @return List of backup code of the user.
      * @throws BackupCodeException If an error occurred while getting backup codes.
      */
-    public static List<String> getBackupCodes(String username) throws BackupCodeException {
+    public static Integer getBackupCodes(String username) throws BackupCodeException {
```

Comment:
Shall we change the method name as it returns backup code count

Reply:
addressed with [https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/3426661b07018f6570da38cf3f2431989adbc5df](https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/3426661b07018f6570da38cf3f2431989adbc5df)